performance-test/src/.env

```diff
@@ -0,0 +1,2 @@
+AWS_ACCESS_KEY_ID = "AKIAJQXJFEFKZZOS4S6A"
```

Comment:
I believe these could moved to environment variables for better security

Reply:
actually, since this repository is open source, by committing these here you exposed thse secret keys to the world, which means we should (quickly!) revoke access to this key in AWS, create a new key and put that to the proper place for it (I think @seanparsons  can give you pointers about how to use the repository which holds our secret keys)